flag/service/collector/issue/issue.go

```diff
@@ -0,0 +1,5 @@
+package issue
+
+type Issue struct {
+	CustomLabels string
+}
```

Comment:
I would prefer to have `Collector` and `Issue` in the same package like `types`. This would be a clean indication those are data types. `collector.Collector` is usually a service type.

Reply:
When you want to have a different approach to the flag handling than we have right now, feel free to provide a PR to `fmt` and get consensus on your desired practise. I do what we have agreed on so far.

Reply:
Ah those are flags. Sorry. All good.